lib/reporters/tap.js

```diff
@@ -118,6 +119,21 @@ class TapReporter {
 		}) + os.EOL);
 	}
 
+	writeLogs(evt, flags = {error: false, title: ''}) {
```

Comment:
In my mind, "logs" here refers to `evt.logs`, but that doesn't quite cover the scenarios in which it's called. What are you referring to? Should this be `writeTapComment()`?

Reply:
I agree the proposed name is better, but since any function inside the reporter uses TAP, I will choose `writeComment`